### PR TITLE
Login Page

### PR DESCRIPTION
**Pull request by atstojanov**
- [ ] Missing: Register, Logout.
- [ ] Must check: Token Expiration.

### Additional context

Add any other context

### Governance 

- [ ] Documentation is added
- [ ] Test cases are added or updated
- [ ] I've read the contributing document https://github.com/cherrytwist/.github/blob/master/CONTRIBUTING.md
- [ ] I've read and understand the Code of Conduct https://github.com/cherrytwist/.github/blob/master/CODE_OF_CONDUCT.md
- [ ] I understand that any contributions or suggestions I made may make it into the actual code. I've read the License 
     https://github.com/cherrytwist/Coordination/blob/master/LICENSE 
     and the contributor license agreement: tba

### CODE REVIEW

File: src/components/LoginPage.tsx [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+});
+
+const initialValues = {
+  username: '',
+  password: '',
+};
+const loginQuery = async (username: string, password: string) => {
+  return await axios
+    .post(
+      '/auth/login',
```
Comment (atstojanov) [inline src/components/LoginPage.tsx:32]: Must be read from configuration

File: src/models/Constantes.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+export const TOKEN_KEY = 'accessToken';
```
Comment (valentinyanakiev) [inline src/models/Constantes.ts:1]: Constants.ts not Constantes.ts - there is a typo

File: src/setupProxy.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  app.use(
+    '/graphql',
+    createProxyMiddleware({
+      target: 'http://localhost:4000',
+      changeOrigin: true,
+    })
+  );
+  app.use(
+    '/auth',
+    createProxyMiddleware({
```
Comment (valentinyanakiev) [inline src/setupProxy.js:13]: This configuration should be dynamic. If the endpoint of Simple Auth Provider is changed in the config - this is not reflected here, and I have a working example:
1. Docker compose the server project - it was changed to map 3003:3002 so Demo and Server project don't collide
2. Try to login with simple auth

It doesn't work and no error message is show, simply nothing happens when 'login' is clicked.

**Review by valentinyanakiev — CHANGES_REQUESTED** [review]
Overall looks good, just fix the typo and create the proxy endpoints from the config. The rest looks OK to me.

**Review by valentinyanakiev — APPROVED** [review]

**Pull request merged**